### DISCUSSION TITLE
Can Tycho source the branding of a feature without a branding plug-in?

### DISCUSSION THREAD

**Discussion by vogella** [eclipse-tycho/tycho#360] [Q&A] [answered]
AFAIK a feature can currently be branded with a reference to an additional plug-in to get the following additional files:

about.html: 
about.ini:
about.properties: 
about.mappings: 
image for a feature

It seems wasteful to create plug-ins only for the these entries. It is possible to configure Tycho to source these files directly from the feature?

**Comment by laeubi** [comment, marked as answer]
First of all, you can just use *any* plugin as a branding plugin (e.g. just choose one of your UI plugins for that purpose), second, only about.ini and image are required.

Beside that, tycho does nothing with the branding itself, its all up to the runtime (e.g. Eclipse IDE) to do anyithing usefull with that, so it might be better to ask at Eclipse Platform if it is possible to place these informations (or add support for it) directly in the feature.

**Reply by vogella** [reply]
Thanks @laeubi, sounds good. So we only would have to update the about dialog to source the files directly from the feature.

**Reply by laeubi** [reply]
Ther is some kind of indirection so FeatureDialog do not need to be updated but the code that maps Feature -> Branding I think `org.eclipse.core.runtime.Platform.getBundleGroupProviders()` is a good starting point.

**Comment by vogella** [comment]
This question can be marked as answered.
